packages/core/src/http-middlewares/after/throw-for-status.js

```diff
@@ -1,12 +1,10 @@
 'use strict';
 
-const { stripQueryFromURL } = require('../../tools/http');
+const errors = require('../../errors');
 
 const throwForStatus = resp => {
   if (resp.status > 300) {
-    const cleanURL = stripQueryFromURL(resp.request.url);
```

Comment:
Looks like `stripQueryFromURL` will be no longer used after this change. If so, let's remove it.